### ISSUE TITLE
[Global / Chrome] "switch" handling in Chrome inconsistent (Chrome bug).

### ISSUE THREAD

**Issue by 2Abendsegler** [2Abendsegler/GClh#1743]
[Global / Chrome] "switch" handling in Chrome inconsistent (Chrome/Tampermonkey bug).

Because the function `is_page(name)` is very central, practically nothing works, if chrome no longer wants to. Or all works, if chrome just wants to. 😲 We have nothing changed here since months. 😅
But who also works with chrome? 🤣

---
Old: That worked before and now only sometimes, yes only sometimes.
```
if (is_page('lists')) {
    ....

function is_page(name) {
    var status = false;
    var url = document.location.pathname;
    switch (name) {
        ...
        case "lists":
            if (url.match(/^\/plan\/lists/)) status = true;
            break;
        ...
    }
    return status;
}
```

---
New: 
```
if (is_page('lists')) {
    ....

function is_page(name) {
    var status = false;
    var url = document.location.pathname;
    switch (name) {
        ...
        case "lists":
            if (url.match(/^\/plan\/lists/)) status = true;
            return status;
        ...
    }
    return status;
}
```

**Comment by 2Abendsegler** [comment]
In addition to the `is_page` shift processing, there are nine more. All have to be rearranged. What a shit.

**Comment by 2Abendsegler** [comment]
We finally got the strange behavior in the Chrome browser. The problem has been going on for weeks. It seems to be a bug in the Chrome browser or possibly a bug between the Chrome browser and Tampermonkey. The strange behavior showed up in such a way that the GClh sometimes did its job and then again not, although exactly the same geocaching page was processed.

The bug did not show itself to all Chrome users. Perhaps it was because some are right-handed and the other left-handed, or the bug was based on the Corona case numbers in the respective district. We do not know it. We only identified the language element that was sometimes correctly and sometimes incorrectly interpreted.

The affected Chrome users will be happy.
Firefox users don't care, they have been able to work without problems for weeks. 

---
merged
